conans/client/tools/env.py

```diff
@@ -30,13 +30,14 @@ def run_environment(conanfile):
 
 
 @contextmanager
-def environment_append(env_vars):
+def environment_append(env_vars, post=False):
```

Comment:
Do we want to add it to the tool and to expose the new behavior to the user?  (no one has asked for it so far)